bin/setup

```diff
@@ -31,6 +36,7 @@ if [[ $USER == 'ubuntu' ]]; then
     echo 'source ~/.virtualenvs/accounting/bin/activate' >> ~/.bashrc
 fi
 
+
```

Comment:
@macornwell Nit: Make whitespace between "sections" (delimited by inline comments) consistent -- there's two empty lines in some cases and a single empty line (which should probably be the default) in others.